### DISCUSSION TITLE
rules: complexity ignore react render

### DISCUSSION THREAD

**Discussion by pisyell** [eslint/eslint#16893] [Q&A] [answered]
<img width="751" alt="image" src="https://user-images.githubusercontent.com/37040920/218915449-6ae720f8-aef7-4e68-a5f1-3565e6a8cb72.png">

how to fix this？i need the complexity config.
This is only a demo, and the nodes may be discontinuous.

**Comment by xxf1110** [comment]
me too

**Comment by Rec0iL99** [comment, marked as answer]
Hi @pisyell and @xxf1110, this is happening because of the `complexity` ESLint rule being set manually, the default is a value of 20. You can fix it by increasing the complexity value which in your case would be:

`complexity: ["error", 12]`

**Tip from the docs:** If you can’t determine an appropriate complexity limit for your code, then it’s best to disable this rule.

More info on this can be found in the [docs](https://eslint.org/docs/latest/rules/complexity).

Thanks for using ESLint.

**Comment by jotaWTW** [comment]
You also can add this comment at the beginning of your code
`/* eslint-disable complexity */`
